src/templates/templates.service.spec.ts

```
@@ -0,0 +1,137 @@
+
+    it('should throw NotFoundException if clause does not exist', async () => {
+      repository.findOne.mockResolvedValue(null);
+
+      await expect(service.findOne('2')).rejects.toBeInstanceOf(NotFoundException);
+    });
+  });
+
+  describe('update', () => {
+    it('should update a clause', async () => {
+      const clause = { id: '1', name: 'A' } as StandardClause;
+      const dto: UpdateStandardClauseDto = { name: 'B' } as UpdateStandardClauseDto;
```

ESLint

[error] Replace `·name:·'B'` with `⏎········name:·'B',⏎·····`

(prettier/prettier)